README.md

```diff
@@ -12,7 +12,13 @@ You can follow our [Get Started](./docs/getting_started.md) guilde to quick star
 
 ## Contributing
 
-Contributions are welcome and greatly appreciated. See [develop guide](./docs/develop_guide.md) for details about Matrixone Operator develop story ideas.
+Contributions are welcome and greatly appreciated. See [develop guide](./docs/dev_guide.md) for details about Matrixone Operator develop story ideas.
+
+## Notice
+- The Operator currently runs on TKE/EKS. GKE, AKC, and other Managed Public Cloud k8s have not been tested.
+- multi-region is not supported yet.
+- The Operator only supported 1 nodes cluster.
```

Comment:
The default cluster size is set to 1 node temporarily, the multi-nodes cluster has not been fully tested.